Cargo.toml

```
@@ -34,7 +34,6 @@ paste = "1.0.2"
 palette = { version = "0.7.6", optional = true }
 serde = { version = "1", optional = true, features = ["derive"] }
 strum = { version = "0.26", features = ["derive"] }
```

The removed strum_macros crate in the line below this was added to ensure that strum correctly picked up the right version of that as there was a conflict with something in itertools a while back. When this was added when strum was only available as 0.26.2, and only the macros package was at 0.26.3, so that dependency was actually necessary. Since then, strum released 0.26.3 which references strum_macros 0.26.3, so the right thing to do here is update the main crate version and delete the macros crate as done below.

```suggestion
strum = { version = "0.26.3", features = ["derive"] }
```